client/modules/IDE/components/Editor.jsx

```diff
@@ -269,15 +269,20 @@ class Editor extends React.Component {
       indent_size: INDENTATION_AMOUNT,
       indent_with_tabs: IS_TAB_INDENT
     };
-
     const mode = this._cm.getOption('mode');
+    const currentPosition = this._cm.doc.getCursor();
     if (mode === 'javascript') {
       this._cm.doc.setValue(beautifyJS(this._cm.doc.getValue(), beautifyOptions));
+      console.log('beautifying');
```

Comment:
could you remove this last `console.log`? otherwise this is working great for me!

Reply:
hey @catarak this is fixed and can be merged now